docs/development/pigs/pig-024.rst

```diff
@@ -0,0 +1,220 @@
+-----------------------------
+
+Contributing to Gammapy can be made in different ways:
+
+- contributing to gammapy source code,
+- contributing to our documentation (rst pages, docstring, gammapy-webpage),
+- contributing to our DevOps environment (CI, GitHub configurations, gammapy-docs, release processes, etc),
+- contributing to our associated projects (gammapy-benchmarks, gammapy-data, gammapy-extra, gammapy-cat, gammapy-recipes),
+- contributing to the maintenance of our repositories (conda-forge, PyPI, Zenodo, HAL, docker-hub, etc) or open hub (openhub, etc),
+- leading the precise tasks identified by the Lead Developers,
+- organizing and leading hands-on sessions or schools,
+- managing our social media accounts,
+- and, de-facto, the members of the Coordination Committee.
+
+Any contribution on these topics renders the user as an official Gammapy contributor.
+
+.. important::
+    We strongly recommend that the contributors use always the same GitHub account, to have a unique
+    `ORCID <https://orcid.org/>`_ identifier and store it into your GitHub user profile, and inform the developers
+    of any change of affiliation.
+
+As Gammapy is an open project, we encourage users to provide feedback on the usage of Gammapy, on features, on issues
+via our communication channels.
+
+Definition of the `Maintainer`
+------------------------------
+
+In the Open Source community, a maintainer is generally mentioned in the metadata associated with a release and has to be mentioned.
+Usually, a software maintainer or package maintainer is one or more people who build source code into a binary package
+for distribution, commit patches, or organize code in a source repository.
+
+For Gammapy, by default, the maintainers are the Lead Developers. If in the future a task is dedicated to the creation
+of a release, the maintainer will be the person in charge of this task.
+
+Releases
+--------
+
+Each Gammapy release should be associated with an updated list of authors. The list of authors is composed by
+people who contributed as listed in the previous sub-section to the current, or in the previous release in
+order to account for long contributions (e.g. management of depreciated features, improvement/debugging of features,
+etc). Any additional author can be inserted into the list for its special contribution when requested by the Gammapy
+Coordination Committee.
+
+The order of the authors is first 'The Gammapy Collaboration', the Lead Developers, a past Lead Developer in case of
+change between two releases, followed by the list of contributors in alphabetical order.
+
+This policy is applied to any release after the LTS V1.0. For the releases made before, a hand-made list is used.
+After the mention of 'The Gammapy Collaboration', the current Lead Developers will be in first, then the past Lead
+Developers, followed by the list of contributors in alphabetical order.
+
+Long Term Support releases
+--------------------------
+
```

Comment:
In this PIG, I have not written that Gammapy will always make a paper for an LTS. Maybe it is out-of-scope of this PIG ?

Reply:
You wrote that people should cite the paper belonging to the latest LTS release, which implies that there always is such a paper

Reply:
We are preparing the paper for v1.0, so this will be there for sure. But indeed if there is no paper v2.0 then people should continue citing the one for v1.0 instead. 

In general a small paper every two years should be possible to write and is fair to developers to collect academic credit as well. But there are thing out of our control so, e.g. maybe the paper is not accepted, so I agree we should rephrase to cite the latest Gammapy paper instead (whichever it is...).

Reply:
>yes, this is what we have foreseen and decided.

I fully agree this is what we should aim for, but I'm not sure we can commit to it? E.g. a paper could be not accepted for publication (it's unlikely but anyway...)

Reply:
the new sentence is :+1: 
`When Gammapy is used for any publication, contribution to conferences or software, authors should properly cite Gammapy. It is asked to cite the DOI of the used Gammapy version as well as the associated paper (e.g. the latest LTS release).`

AND, few lines after:
`This citation scheme with these **two references** will be given on our web pages.
`